src/cmd/restart.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+*/
```

Comment:
Is this intentional?

Reply:
Missed it! Thanks for pointing this out :)